db/structure.sql

```diff
@@ -1752,6 +1755,8 @@ INSERT INTO "schema_migrations" (version) VALUES
 ('20190312104101'),
 ('20190325082640'),
 ('20190326113949'),
-('20190609165906');
+('20190609165906'),
+('20190730133328'),
+('20190731151806');
```

Comment:
Should these lines be in this commit? And if so should there be three entries here? I only count two regular db migrations...

Reply:
This PR is out of sync with PR No. 3 because I sneakily snuck in another migration for some additional columns in the new warehouse_case_reports table. Will be amended accordingly.